common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1314,154 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+		heavy_infantry = 0.05
+		light_infantry = 0.05
+		archers = 0.05
+		light_cavalry = 0.05
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion = fel_group
```

Comment:
Syntax error. `fel_group` is religion group and needs `religion_group` trigger.
```suggestion
				religion_group = fel_group
```

Reply:
good point, done